rmw_fastrtps_cpp/package.xml

```diff
@@ -32,8 +31,6 @@
   <build_export_depend>rcutils</build_export_depend>
   <build_export_depend>rmw_fastrtps_shared_cpp</build_export_depend>
   <build_export_depend>rmw</build_export_depend>
-  <build_export_depend>rosidl_generator_c</build_export_depend>
-  <build_export_depend>rosidl_generator_cpp</build_export_depend>
```

Comment:
No rational for these changes.

Reply:
Still pending.

Reply:
`rosidl_generator_c` is declared as `<depend>` https://github.com/ros2/rmw_fastrtps/pull/351#issuecomment-595683417

rosidl_generator_cpp is not been used

Reply:
So you argue that the dependency was unnecessary - independent of this PR.

In the future I can only highly recommend to separate changes which are unrelated into separate PRs. This kind of mixture (which is the case in several of the related PRs) is making the changes harder to read and review and therefore increase the probability that some dependency is actually incorrectly changed. Also please comment on feedback like this "inline" (like now) so that the explanation is visible. The referenced comment is not related to this specific feedback and doesn't at all answer the question. That is probably why this was sitting for 7 days without further update.

Reply:
Yes, as you can see in the [changelog](https://github.com/ros2/rmw_fastrtps/blob/master/rmw_fastrtps_cpp/CHANGELOG.rst)

> Remove unnecessary dependency on rosidl_generator_cpp (#161)

or if you try to find some related headers https://github.com/ros2/rmw_fastrtps/search?q=rosidl_generator_cpp%2F&unscoped_q=rosidl_generator_cpp%2F

`rosidl_generator_cpp` is not there anymore

Reply:
> Yes, as you can see in the changelog

I am not understanding: how is a changelog entry from 2+ years ago sufficient to justify this? Obviously it had been readded since then for some reason.

Reply:
I was trying to say that maybe this dependency in the `package.xml` is a old fashion line, which is living there because it was not removed.

I can create a separate PR to remove this line if this will clean up this PR